source/detail/serialization/serialisation_helpers.hpp

```diff
@@ -0,0 +1,96 @@
+#ifndef XLNT_DETAIL_SERIALISATION_HELPERS_HPP
+#define XLNT_DETAIL_SERIALISATION_HELPERS_HPP
```

Comment:
this was moved out of xlsx_consumer.cpp as a bit of cleanup from #421
I was orginally thinking about making similar changes to the serialiser but decided against it. This remains as a separate file as it probably always should have been (in general, consumers is too large already)